packages/adblocker/src/engine/domains.ts

```diff
@@ -13,15 +13,17 @@ import { binLookup, hasUnicode, HASH_INTERNAL_MULT } from '../utils.js';
 
 export class Domains {
   public static parse(
-    value: string,
+    value: string | Set<string>,
```

Comment:
should it be `string[]` instead? one type, no string

Reply:
We don't pass `string[]` type. Alternatively, we can refactor them to accept `string[]` without a delim. However, we still need to pass a delim to the object as we need to care about `toString`.